libs/src/utils/signatures.ts

```diff
@@ -1,23 +1,24 @@
-const Utils = require('./utils.js')
-const { ecsign, toBuffer } = require('ethereumjs-util')
-const { pack } = require('@ethersproject/solidity')
+import Utils from './utils'
```

Comment:
it's a weird one where the utils here is the utils.ts file which default exports Utils. the `utils/index.ts` file (which is in this directory) takes that default export and rexports it as a named export for files outside this utils directory. We can't import the index.ts file in signatures since `index.ts` exports signatures, which would cause a circular import. Ideally `utils.ts` just exports Utils as a named export for consistency, going to update that!

Reply:
🤯